packages/calcite-components/src/components/block/block.stories.ts

```diff
@@ -122,23 +122,23 @@ export const darkModeRTL_TestOnly = (): string => html`
   <calcite-block
     heading="Heading"
     description="description"
-    open
+    expanded
     collapsible
```

Comment:
After seeing the renamed prop in context, I think we should deprecated `collapsible` in favor of `expandable`. We can tackle this separately to allow for some initial discussion.

Reply:
[[Block] Deprecate collapsible in favor of expandable](https://github.com/Esri/calcite-design-system/issues/11671#top)#11671